City-of-Helsinki/events-helsinki-ui
# Th-908: Bottom banner

## Description :sparkles:
Add bottom banner to front page. Also refactored topBanner to use Landingpage > topBanner fields.

## Issues :bug:
https://helsinkisolutionoffice.atlassian.net/browse/TH-908
## Testing :alembic:
start server locally and:
- Change bottom banner from cms: https://cms.test.kuva.hel.ninja/admin/pages/60/edit/ see how it changes on front page:  http://localhost:3000/fi/home 
- Try to change some other page: https://cms.test.kuva.hel.ninja/admin/pages/81/edit/ and preview the change:
http://localhost:3000/fi/home/81

## Screenshots :camera_flash:

## Additional notes :spiral_notepad:
